src/site/content/en/newsletter/index.njk

```diff
@@ -5,6 +5,8 @@ permalink: false
 description: Stay up to date with the latest about web platform.
 date: 2020-03-13
 draft: true
+eleventyComputed:
+  noindex: "{{ paged.index > 0 }}"
```

Comment:
Why not boolean here?

Reply:
I can't get the `noindex` value be a `boolean` when computed by the njk template. I can only get it to be processed as a `string`. If I did it in a JS file I could do a `boolean`, but because we're extending a NJK template we can't also do a 11tydata file.